hypha/apply/activity/templates/emails/activity_summary.md

```diff
@@ -0,0 +1,29 @@
+{% load humanize i18n %}
+
+{% trans "Dear" %},
+
+{% blocktranslate with total_count_apnumber=total_count|apnumber count counter=total_count %}
+You have {{ total_count_apnumber }} new activity:
+{% plural %}
+You have {{ total_count_apnumber }} new activities:
+{% endblocktranslate %}
+
+{% if submissions %}**{% trans "Submissions" %}**
+{% for msg in submissions %}
+- {{ msg.content_markdown }} • {{ msg.event.when }}{% endfor %}{% endif %}
```

Comment:
After each "•" there is a nonbreaking space character (0xa0). I think that should be a normal space.

Reply:
Fixed in https://github.com/HyphaApp/hypha/pull/2991/commits/d67ffadaccfb2c3f0f604d987ba6f16ab1916efc